src/server/routes/match.route.js

```diff
@@ -4,8 +4,8 @@ import * as match from '../controllers/match.controller';
 const router = express.Router();
 
 router.post('/request', match.requestMentoring);
+router.post('/response', match.responseMentoring);
 
-router.get('/accept', match.acceptRequest);
-router.get('/reject', match.rejectRequest);
+router.get('/activity', match.getMyActivity);
```

Comment:
Leave a comment at the top of each block (line 5 and line 8) that these are the POST/GET methods to make it obvious why we have 2 diff blocks.